### PR TITLE
Java persistence init

### PR DESCRIPTION
**Pull request by MrLoge**

### CODE REVIEW

**Comment by MrLoge** [comment]
@DennisAhaus Can you please review my PullRequest?

File: README.md [hunk 1]
```diff
@@ -1 +1,186 @@
...
+#Java Persistence API - general project for the OrgApp
+
+This project consists of the dependencies for JPA and Hibernate. 
+
+The Maven dependency for this project is: 
+
+```Maven
+    <dependency>
+	  <groupId>com.orgapp.jpa</groupId>
+	  <artifactId>orgappJPA</artifactId>
```
Comment (DennisAhaus) [inline README.md]: Please rename the artifact to lowercase letters

File: README.md [hunk 2]
```diff
@@ -1 +1,186 @@
...
+	  <version>1.0-SNAPSHOT</version>      
+    </dependency>  
+```
+
+##Usage of the JPA
+
+In every sub-project, a persistence.xml document must be created. The document has to be stored in the resources/META-INF directory.
```
Comment (DennisAhaus) [inline README.md]: `The document has to be stored`--> `The file has to be stored`

File: README.md [hunk 3]
```diff
@@ -1 +1,186 @@
...
+	  <artifactId>orgappJPA</artifactId>
+	  <version>1.0-SNAPSHOT</version>      
+    </dependency>  
+```
+
+##Usage of the JPA
+
+In every sub-project, a persistence.xml document must be created. The document has to be stored in the resources/META-INF directory. 
+
+The document specifies the so called persistence-units which can be referenced from code using an EntityManagerFactory.
```
Comment (DennisAhaus) [inline README.md]: `The document ...` --> `The file ...`

File: README.md [hunk 4]
```diff
@@ -1 +1,186 @@
...
+  ~
+  ~ License: GNU Lesser General Public License (LGPL), version 2.1 or later.
+  ~ See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+  -->
+<persistence xmlns="http://java.sun.com/xml/ns/persistence"
+             xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+             xsi:schemaLocation="http://java.sun.com/xml/ns/persistence http://java.sun.com/xml/ns/persistence/persistence_2_0.xsd"
+             version="2.0">
+
+    <persistence-unit name="org.hibernate.orgapp.jpa">
```
Comment (DennisAhaus) [inline README.md]: Please remove all `hibernate` words or dependencies regarding to own names

File: README.md [hunk 5]
```diff
@@ -1 +1,186 @@
...
+</persistence>
+
+```
+
+###Persistence-Unit
+
+The persistence unit must carry a unique name:
+
+```xml
+<persistence-unit name="org.hibernate.orgapp.jpa">
```
Comment (DennisAhaus) [inline README.md]: Remove `hibernate`

File: README.md [hunk 6]
```diff
@@ -1 +1,186 @@
...
+
+```java
+
+	private EntityManagerFactory entityManagerFactory;
+
+	@Override
+	protected void setUp() throws Exception {
+		// like discussed with regards to SessionFactory, an EntityManagerFactory is set up once for an application
+		// 		IMPORTANT: notice how the name here matches the name we gave the persistence-unit in persistence.xml!
+		entityManagerFactory = Persistence.createEntityManagerFactory( "org.hibernate.orgapp.jpa" );
```
Comment (DennisAhaus) [inline README.md]: Remove `hibernate`

File: README.md [hunk 7]
```diff
@@ -1 +1,186 @@
...
+		this.title = title;
+    }
+}
+```
+
+
+##Further information
+Interesting links with more detailed information about JPA and how to configure the persistence.xml:
+
+https://docs.jboss.org/hibernate/entitymanager/3.5/reference/en/html/configuration.html
```
Comment (DennisAhaus) [inline README.md]: Please explain why we need the jboss hibernate docs here

File: pom.xml [hunk 8]
```diff
@@ -2,8 +2,8 @@
   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
  
-  <groupId>com.orgapp.app</groupId>
-  <artifactId>OrgApp</artifactId>
+  <groupId>com.orgapp.jpa</groupId>
+  <artifactId>orgappJPA</artifactId>
```
Comment (DennisAhaus) [inline pom.xml]: Rename to lowercase

File: resources/META-INF/persistence.xml [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+  ~
+  ~ License: GNU Lesser General Public License (LGPL), version 2.1 or later.
+  ~ See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+  -->
+<persistence xmlns="http://java.sun.com/xml/ns/persistence"
+             xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+             xsi:schemaLocation="http://java.sun.com/xml/ns/persistence http://java.sun.com/xml/ns/persistence/persistence_2_0.xsd"
+             version="2.0">
+
+    <persistence-unit name="org.hibernate.orgapp.jpa">
```
Comment (DennisAhaus) [inline resources/META-INF/persistence.xml]: remove `hibernate`

**Comment by DennisAhaus** [comment]
There is also a problem with running tests. If you look at travis-ci console output there are no tests running.

**Comment by MrLoge** [comment]
> There is also a problem with running tests 

The general jpa project doesn't have any test-cases anymore because I only wanted to keep the pom.xml and an example persistence.xml

**Comment by MrLoge** [comment]
@DennisAhaus I have changed the files according to your claims

**Pull request merged**